content/references/rippled-api/transaction-formats/transaction-types/trustset.md

```diff
@@ -31,9 +31,8 @@ Create or modify a trust line linking two accounts.
 | `LimitAmount`.`currency` | String    | (Amount.currency) | The currency to this trust line applies to, as a three-letter [ISO 4217 Currency Code](http://www.xe.com/iso4217.php) or a 160-bit hex value according to [currency format](currency-formats.html). "XRP" is invalid. |
 | `LimitAmount`.`value`    | String    | (Amount.value)    | Quoted decimal representation of the limit to set on this trust line. |
 | `LimitAmount`.`issuer`   | String    | (Amount.issuer)   | The address of the account to extend trust to. |
-| `QualityIn`              | Number    | UInt32            | _(Optional)_ Value incoming balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` is shorthand for treating balances at face value. |
-| `QualityOut`             | Number    | UInt32            | _(Optional)_ Value outgoing balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` is shorthand for treating balances at face value. |
-
+| `QualityIn`              | Number    | UInt32            | _(Optional)_ Value incoming balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` treats balances at face value. A value of `1` is recommended.  |
+| `QualityOut`             | Number    | UInt32            | _(Optional)_ Value outgoing balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` treats balances at face value. Set this value higher than `QualityIn` to charge a percentage for Ripples to this trust line. For example, if `QualityIn` is `1` and `QualityOut` is `1.01`, the charge is 1%. |
```

Comment:
The example here is also not quite right for the same as with the previous line.